jest.a11y.config.js

```diff
@@ -1,12 +1,14 @@
 module.exports = {
   roots: ['<rootDir>/src/test/a11y'],
-  testRegex: "(/src/test/.*|\\.(test|spec))\\.(ts|js)$",
-  testEnvironment: "node",
+  "testRegex": "(/src/test/.*|\\.(test|spec))\\.(ts|js)$",
+   "moduleFileExtensions": [
+    "ts",
+    "js"
+  ],
+  "testEnvironment": "node",
   transform: {
     '^.+\\.ts?$': 'ts-jest',
   },
-  moduleFileExtensions: ['ts', 'js'],
-  setupFiles: ["<rootDir>/jestEnvVars.js"],
-  // pa11y recommended default
-  testTimeout: 30000,
```

Comment:
why are you removing this?

Reply:
No, I didn't remove it :) I just changed 4 files which are linked to tests and merged pre-release branch into this branch.

Reply:
done